CHANGELOG.md

```diff
@@ -33,9 +34,9 @@
 - Fix initcontainer script (#202) 
 - Refine Fluentd CRs status (#225)
 - Fix ci and make the repository importable and downloadable (#229)
-- Fix codegen && add support for verifying codegen. (#234 #238)
-- Fix helm && Optimize helm. (#236 #245 #246)
-
+- Fix codegen && add support for verifying codegen (#234 #238)
+- Fix helm && Optimize helm (#236 #245 #246)
+- fix fluentbit operator not support Time_Key_Nanos field bug (#250)
```

Comment:
fix fluentbit operator not support Time_Key_Nanos field bug (#250)
=>
Add Time_Key_Nanos field  (#250)